labs/firepower-restapi-106/firepower-restapi-106.json

```diff
@@ -30,9 +30,11 @@
         "title": "Cisco Security REST API"
     }],
     "files": [{
-        "title": "1.md"
+        "file": "1.md",
+        "title": "FMC - REST - API - 106"
     }, {
-        "title": "2.md"
+        "file": "2.md",
+        "title": "FMC REST API Objects"
```

Comment:
Make sure the title matches what's in the Markdown file - it seems like it would be objects, lowercase.

Reply:
No, actually, in the titles, most are capitalized. Let's stay consistent...